tests/extension/oneapi_sub_group_mask/sub_group_mask_insert_bits.cpp

```diff
@@ -34,7 +34,7 @@ void get_bits(sycl::marray<T, nElements> &out) {
 
 template <typename T>
 struct check_result_insert_bits {
-  bool operator()(const sycl::ext::oneapi::sub_group_mask &sub_group_mask,
```

Comment:
I think we should keep `const` here or pass by value. This operator is supposed to **check** mask properties and it might lead to unexpected bugs if it changes it.
Please, apply to all checkers.

Reply:
Changed to pass by value.